bindata/network/openshift-sdn/controller.yaml

```diff
@@ -119,6 +119,7 @@ spec:
 {{- if not .ExternalControlPlane }}
       nodeSelector:
         node-role.kubernetes.io/master: ""
+        kubernetes.io/os: "linux"
```

Comment:
Should `kubernetes.io/os: "linux"` selector be limited only to deployments without `ExternalControlPlane`? I don't think it should

Reply:
true